devices/atlantic.js

```diff
@@ -0,0 +1,157 @@
+            assert(typeof value === 'boolean');
+            await entity.write('hvacFanCtrl', {0x1000: {value: value ? 1 : 0, type: 0x10}}, {manufacturerCode: 0x125b});
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === thermostatPositions.swing) {
+                await entity.write('hvacThermostat', {0x4274: {value: 1, type: 0x10}}, {manufacturerCode: 0x125b});
+            } else {
+                await entity.write('hvacThermostat', {0x4274: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                await entity.write('hvacThermostat', {0x4273: {value: index, type: 0x30}}, {manufacturerCode: 0x125b});
+            }
+            return {state: {ac_louver_position: value}};
+        },
+    },
+    energy_mode: {
+        key: ['energy_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(energyMode));
+            const index = energyMode[value];
+            switch (index) {
+            case energyMode.eco:
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                await entity.write('hvacThermostat', {'programingOperMode': 4});
+                break;
+            case energyMode.normal:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}}, {manufacturerCode: 0x125b});
+                break;
+            case energyMode.powerful:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                await entity.write('hvacThermostat', {0x4270: {value: 1, type: 0x10}}, {manufacturerCode: 0x125b});
+                break;
+            }
+            return {state: {energy_mode: value}};
+        },
+    },
+    thermostat_programming_operation_mode: {
+        key: ['programming_operation_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            await tz.thermostat_programming_operation_mode.convertSet(entity, key, value, meta);
+            return {state: {programingOperMode: value}};
+        },
+        convertGet: tz.thermostat_programming_operation_mode.convertGet,
+    },
+    thermostat_occupied_cooling_setpoint: {
+        key: ['occupied_cooling_setpoint'],
+        options: [exposes.options.thermostat_unit()],
+        convertSet: async (entity, key, value, meta) => {
+            await tz.thermostat_occupied_cooling_setpoint(entity, key, value, meta);
+            return {state: {occupied_cooling_setpoint: value}};
```

Comment:
This can be moved directly into `tz.thermostat_occupied_cooling_setpoint`, then this custom converter can be removed.